docs/dev/graphics-stack.md

```diff
@@ -156,17 +156,17 @@ Additionally, information about driver development:
 
 ##### SNA
 
-May causes segmentation fault if enabled. If so, just disable SNA and you can copy attached 20-intel-uxa.conf to /etc/X11/xorg.conf.d/.
+may cause segmentation faults if enabled. If so, just disable SNA and you can copy the attached 20-intel-uxa.conf to /etc/X11/xorg.conf.d/.
```

Comment:
Why the lower-case `m` change in `may cause`?

Reply:
Because "May cause segmentation faults if enabled." is no full sentence.
Combined with the header it gives
"SNA
may cause segmentation faults if enabled." which is a full sentence. Maybe repeating SNA at the beginning of the sentence is better, eg.
"SNA may cause segmentation faults if enabled."
What do you think?

Reply:
> "SNA may cause segmentation faults if enabled."
> What do you think?

This one make sense to me.

Reply:
Done